databricks/conftest.py

```diff
@@ -96,7 +96,7 @@ def add_db(doctest_namespace):
     doctest_namespace["db"] = db_name
 
 
-@pytest.fixture(autouse=os.getenv("KOALAS_USAGE_LOGGER", None) is not None)
+@pytest.fixture(autouse=os.getenv("KOALAS_USAGE_LOGGER", "") != "")
```

Comment:
Hmm .. why do we need this though? I thought we can just `unset KOALAS_USAGE_LOGGER`.

Reply:
The variable is always set in CI with conda.
https://github.com/databricks/koalas/blob/bc51b4b06c8db7eca60a3d8f8b31aa4db98642b1/.github/workflows/master.yml#L131
I just wanted to ignore it.

Reply:
It's not a big deal. Let me just revert it.

Reply:
Ah, got it. it should be fine.

Reply:
ah, ok, let's have it.